optimize/self-managed/optimize-deployment/migration-update/3.9-to-3.10.md

```diff
@@ -9,6 +9,24 @@ To update Optimize to version 3.10, perform the steps in the [migration and upda
 
 The update to 3.10.x can be performed from any 3.9.x release.
 
+Here you will find information about:
+
+- Limitations
+- Known issues
+- Changes in the supported environments
+- Any unexpected behavior of Optimize (for example, due to a new feature)
+- Changes in translation resources
+
+## Limitations
+
+No known issues at the moment.
+
+## Known issues
+
+## Changes in the supported environments
+
+## Any unexpected behavior of Optimize (for example, due to a new feature)
```

Comment:
I don't think we need these subsections if there is nothing to include, but the bulleted list above is enough

Reply:
word